### DISCUSSION TITLE
Keycloak integration

### DISCUSSION THREAD

**Discussion by kimizumi1** [inventree/InvenTree#5744] [Q&A]
Hi!
I noticed InvenTree has allauth enabled by default. Also documentation shows that Keycloak is supported.
Sadly I have no idea how that should work. I've tried to search information of it but with no success.
I have already a working Keycloak instance running. 

Has someone this in use or does someone know how to get it working?
Would be a really nice feature!

**Comment by SchrodingersGat** [comment]
@kimizumi1 as per our [SSO documentation](https://docs.inventree.org/en/stable/settings/SSO/#configuration-file) you will have to refer to the `django-allauth` docs for information on setting up a particular provider. Our docs run through a simple example for google integration, which you can take as a starting point.

There are specific docs for keycloak on the [allauth docs page](https://docs.allauth.org/en/latest/socialaccount/providers/keycloak.html)
